spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/TaskValidationProblem.java

```diff
@@ -19,18 +19,18 @@
 /**
  * After parsing a composed task definition from a DSL string, the validation visitor may optionally run.
```

Comment:
Currently the only things found as validation problems would occur on composed tasks but I think it can be a more generally applicable thing (to simple tasks too). So I'll change that.